Tests/XCUITests/SearchTest.swift

```diff
@@ -197,8 +197,10 @@ class SearchTests: BaseTestCase {
         // Select some text and long press to find the option
         app.webViews.staticTexts["cloud"].press(forDuration: 1)
         // Click on the > button to get to that option only on iPhone
-        if !iPad() {
-            app.menuItems["show.next.items.menu.button"].tap()
+        while !app.collectionViews.menuItems["Search with Firefox"].exists {
```

Comment:
Same as the change in `FindInPageTest`. Even if "Search with Firefox" is located in a different place in the context menu due to device size or UI change, we don't have to update the test.